app/src/main/java/com/example/firstandroid/MainActivity.kt

```diff
@@ -92,7 +112,8 @@ class MainActivity : ComponentActivity() {
                             if (isUpdateDialogOpen) {
                                 UpdateBathroomDialog(
                                     onDismissRequest = { onBackPressed() },
-                                    viewModel = updateBathroomViewModel
+                                    viewModel = updateBathroomViewModel,
```

Comment:
I would split this across multiple lines. And I would use named arguments as it would be easy to get the order wrong.